### PR TITLE
Updating NPM to publish newly as "ids-enterprise"

### PR DESCRIPTION
**Pull request by clepore**
Explain the **details** for making this change. What existing problem does the pull request solve?

So we can have an "open" version of `ids-enterprise` going forward

**ALSO** fixes a bug where icons don't show on the demo `app/`

### CODE REVIEW

**Comment by clepore** [comment]
- [ ] I think we need to add a READ.me to the publish folder that will show on NPM.

File: publish/package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "ids-enterprise",
   "slug": "ids-enterprise",
-  "version": "4.6.0-alpha.2",
+  "version": "4.6.0",
```
Comment (tmcconechy) [inline publish/package.json]: Should we/when should we update this to 4.7.0-dev or -rc or something?

Comment (clepore) [reply publish/package.json]: I don't think we should until we are ready to release it.

Comment (clepore) [reply publish/package.json]: Actually, nm. I think you're right.

**Pull request merged**